website/js/util.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
-var dateFormat = (date) => { let d = new Date(date); return `${months[d.getMonth()]} ${d.getDate()}, ${d.getFullYear()}`; };
```

Comment:
Changing this will affect the way the data is displayed on the previous releases page (see http://brcaexchange.org/releases, would turn e.g. `October 6, 2016` into `6 October 2016`). Is this intentional? 

We could also use a library like Moment.js or Date.js or any other number of other libraries to handle this for us along with other date formatting issues we might hit -- common js problem.